src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/BackupConfigurationIT.kt

```diff
@@ -0,0 +1,110 @@
+
+import com.atlassian.performance.tools.dockerinfrastructure.api.browser.DockerisedChrome
+import com.atlassian.performance.tools.dockerinfrastructure.api.jira.Jira
+import com.atlassian.performance.tools.dockerinfrastructure.api.jira.JiraCoreFormula
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
+import com.atlassian.performance.tools.jiraactions.api.SET_UP
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.action.Action
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.measure.output.CollectionActionMetricOutput
+import com.atlassian.performance.tools.jiraactions.api.memories.User
+import com.atlassian.performance.tools.jiraactions.api.memories.UserMemory
+import com.atlassian.performance.tools.jiraactions.api.page.isElementPresent
+import com.atlassian.performance.tools.jiraactions.api.page.wait
+import com.atlassian.performance.tools.jiraactions.api.w3c.DisabledW3cPerformanceTimeline
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.openqa.selenium.By
+import org.openqa.selenium.remote.RemoteWebDriver
+import org.openqa.selenium.support.ui.ExpectedConditions.elementToBeClickable
+import java.time.Clock
+import java.util.*
+
+class BackupConfigurationIT {
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    /**
+     * During the test, you can connect to WebDriver by the VNC viewer.
+     * DockerisedChrome opens the port 5900 so executing command
+     * docker ps
+     * will show you which port on your host is open and mapped to 5900 on the container.
+     * The default password is `secret`.
+     */
+    @Test
+    fun shouldDisableBackups() {
+        val version = System.getenv("JIRA_SOFTWARE_VERSION") ?: "8.0.0"
+        logger.info("Testing Jira $version")
+        val metrics = mutableListOf<ActionMetric>()
+        val actionMeter = ActionMeter(
+            virtualUser = UUID.randomUUID(),
+            output = CollectionActionMetricOutput(metrics),
+            clock = Clock.systemUTC(),
+            w3cPerformanceTimeline = DisabledW3cPerformanceTimeline()
+        )
+        val user = User("admin", "admin")
+
+        JiraCoreFormula.Builder()
+            .version(version)
+            .build()
+            .provision()
+            .use { jira ->
+                DockerisedChrome().start().use { browser ->
+                    val driver = browser.driver
+                    val webJira = WebJira(
+                        driver,
+                        jira.getUri(),
+                        user.password
+                    )
+                    goToServices(driver, jira)
+                    addBackupService(driver)
+
+                    TestingSetup(webJira, actionMeter).run()
```

Comment:
We can make it even simpler:
```suggestion
                    webJira.configureBackupPolicy().delete()
```
and nuke the `TestingSetup`